modules/http2/pom.xml

```diff
@@ -128,6 +128,36 @@
             <version>2.18.3</version>
             <scope>test</scope>
         </dependency>
-
     </dependencies>
+
+    <profiles>
```

Comment:
Possible to use alternatives, but default is JDK version, not npn.
It is not even a breaking change, because the bootstrap was created to fill the gap for JDK8 versions without ALPN support and it is an **old** copy of Sun jsse classes.